packages/patternfly-3/patternfly-react/src/components/LoginPage/LoginPage.stories.js

```diff
@@ -94,7 +94,7 @@ const createProps = () => {
           href: '#',
           onClick: e => storyAction(e, 'Forgot password was clicked')
         },
-        disableSubmit: false,
+        disableSubmit: true,
```

Comment:
maybe remove it from the props here,
as it would be true by default on Login Page With Validation ?

Reply:
Removed and I've added `disableSubmit: true` to default props